conf/query/lucene/indexing_configuration.xml

```diff
@@ -179,6 +179,18 @@ https://wiki.apache.org/jackrabbit/IndexingConfiguration
         <property isRegexp="true" nodeScopeIndex="false">.*:.*</property>
     </index-rule>
 
+    <index-rule nodeType="website:blog">
+        <property>website:title</property>
+        <property>website:summary</property>
+        <property>website:shortsummary</property>
+        <property>website:seosummary</property>
+        <property>website:section</property>
+        <property>website:leadparagraph</property>
+        <property>website:backstory</property>
+
+        <property isRegexp="true" nodeScopeIndex="false">.*:.*</property>
+    </index-rule>
+
```

Comment:
what does it actually do?

Reply:
The indexing-configuration allows to define which doc type/fields are to be included in the search.  The last line excludes everything else (so only index the fields defined).

Reply:
(y)